### ISSUE TITLE
Segmentation fault: 11

### ISSUE THREAD

**Issue by anosiwang** [GanymedeNil/srt2fcpxml#18]
Hello there!

I attempted to use version 2.0.1 and encountered an error when I attempted to run it. Specifically, I received a "Segmentation fault: 11" message. Could you please provide some insight into what might be causing this issue?

Thx a lot.

**Comment by GanymedeNil** [comment]
Can you provide the system environment for running the tool? Also, could you share how you used the tool and provide the complete command used to run it?

**Comment by anosiwang** [comment]
Here is my system info: MacBook Pro 16-inch, 2021, with an Apple M1 Max and 64 GB of memory. I'm running macOS 13.2.1 (22D68).

I downloaded the pre-compiled files for version v2.0.1 from the Releases page and successfully gave the file execute permission using the command `chmod +x /Users/UserName/Downloads/srt2fcpxml`. I then attempted to use the same command I've used before, like so:
`(base) MacBook-Pro:~ UserName$ /Users/UserName/Downloads/srt2fcpxml -fd 30 -srt /Users/UserName/Downloads/sub.en_US.srt`

Oddly enough, the previous versions (v2.0.0 and v1.0.2) worked without any issues, but this newest version isn't functioning properly.
